subsys/net/lib/aws_fota/Kconfig

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2018 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menuconfig AWS_JOBS_FOTA
+	bool "AWS Jobs FOTA library"
+	select AWS_JOBS
+	select JSON_LIBRARY
+
+if AWS_JOBS_FOTA
+
+config CLIENT_ID_MAX_LEN
```

Comment:
Copy-paste from aws_jobs?

Reply:
A lot of the files are a bit copy-paste from different sources. I'll fix this one up soon.

Reply:
Should be fixed now.